sqlglot/dialects/tsql.py

```diff
@@ -370,6 +370,17 @@ def _timestrtotime_sql(self: TSQL.Generator, expression: exp.TimeStrToTime):
     return sql
 
 
+def _parse_datetrunc(args: t.List) -> exp.TimestampTrunc:
+    unit = seq_get(args, 0)
+    this = seq_get(args, 1)
```

Comment:
Since both of them are required in T-SQL we could index them directly here, `seq_get` can get annoying with mypy

Reply:
Scratch that, my bad